src/Calculator.tsx

```diff
@@ -98,7 +98,7 @@ class Calculator extends React.Component {
 
               <table>
                 <tr>
-                    <td>Accuracy:</td><td>{100*((this.state.tP + this.state.tN) / this.getTotal()).toFixed(2)}%</td>
+                    <td>Accuracy:</td><td>{(100*(this.state.tP + this.state.tN) / this.getTotal()).toFixed(2)}%</td>
```

Comment:
> src/Calculator.tsx:98:48 - error TS2363: The right-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.

This one's obvious, you accidentally multiplied by 100 after converting to string, instead of before.